exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,15 @@
 let name = "Daniel";
 let danielsRole = "mentor";
 
+function checkRole(name,danielsRole) {
+  if (danielsRole === "mentor")
+  return `Hi, I'm ${name}, I'm a ${danielsRole}.`;
+  else if (danielsRole ==="student")
+  return `Hi, I'm ${name}, I'm a ${danielsRole}.`;
+
+}
+console.log(checkRole("Daniel","mentor"));
```

Comment:
great function and nice use of interpolation! Remember when you call the function you can use the variables defined on lines 9 & 10, e.g.
```
console.log(checkRole(name, danielsRole));
```

Reply:
Thanks,Ellie.